tests/features/api.feature

```diff
@@ -0,0 +1,283 @@
+      And the "oe_call_tenders_deadline[0][value][year]" field should contain "2020"
+      And the "oe_call_tenders_deadline[0][value][hour]" field should contain "10"
+      And the "oe_call_tenders_deadline[0][value][minute]" field should contain "0"
+      And the "Reference" field should contain "My Reference"
+      And the "Body text" field should contain "My Body text"
+      And the "Teaser" field should contain "My teaser"
+      And the "oe_departments[0][target_id]" field should contain "Asian Development Bank (http://eurovoc.europa.eu/6336)"
+
+    @javascript
+    Scenario: Test OrganisationContentContext and BeforeParseEntityFields alterations are done.
+      Given I am logged in as a user with the "edit any oe_organisation content, access content, view published skos concept entities" permission
+
+      And the following General Contact entity:
+        | Name               | A general contact                                                                            |
+        | Address            | country_code: HU - locality: Budapest - address_line1: General contact 1 - postal_code: 1011 |
+        | Email              | general@example.com                                                                          |
+        | Phone number       | +32477792933                                                                                 |
+        | Social media links | uri: http://instagram.com - title: Instagram - link_type: instagram                          |
+
+      And the following image:
+        | name              | file            |
+        | Image placeholder | example_1.jpeg  |
+
+      And the following "Organisation" Content entity:
+        | Title             | My Organisation             |
+        | Introduction      | My Introduction             |
+        | Body text         | My Body text                |
+        | Acronym           | My acronym                  |
+        | Alternative title | My Alternative Organisation |
+        | Contact           | A general contact           |
+        | EU organisation   | Arab Common Market          |
+        | Organisation type | eu                          |
+        | Logo              | Image placeholder           |
+        | Teaser            | My teaser                   |
+
+      When I visit node "My Organisation" edit page
+      Then I should see the text "My Organisation"
+      And the "Introduction" field should contain "My Introduction"
+      And the "Body text" field should contain "My Body text"
+      And the "Acronym" field should contain "My acronym"
+      And the "Alternative title" field should contain "My Alternative Organisation"
+      And I should see the text "A general contact"
+      And the "oe_organisation_eu_org[0][target_id]" field should contain "Arab Common Market (http://eurovoc.europa.eu/1793)"
+      And the "oe_organisation_org_type" field should contain "eu"
+      And the "Teaser" field should contain "My teaser"
```

Comment:
Let's update the node with a non eu organisation type and check the value was set